rollup.config.js

```diff
@@ -1,31 +1,15 @@
-import babel from 'rollup-plugin-babel'
+import babel from '@rollup/plugin-babel'
 import commonjs from '@rollup/plugin-commonjs'
+import pkg from './package.json'
 import replace from '@rollup/plugin-replace'
 import resolve from '@rollup/plugin-node-resolve'
 import { terser } from 'rollup-plugin-terser'
-import pkg from './package.json'
 
 // Use peerDependencies as rollup external
 // https://rollupjs.org/guide/en/#peer-dependencies
 const { peerDependencies } = pkg
 const external = Object.keys(peerDependencies)
 
-// Explicitly specify unresolvable named exports.
-// https://github.com/rollup/plugins/tree/master/packages/commonjs#namedexports
-const namedExports = {
-  'node_modules/react-is/index.js': ['isForwardRef'],
-  'node_modules/prop-types/index.js': [
-    'array',
-    'arrayOf',
-    'func',
-    'number',
-    'object',
-    'oneOf',
-    'oneOfType',
-    'string'
-  ]
-}
```

Comment:
Was this no longer needed in modern versions of the rollup commonjs plugin?

Reply:
You can refer to these two PRs for more detail:

https://github.com/rollup/rollup/pull/3295
https://github.com/rollup/plugins/pull/149

In short, with `syntheticNamedExports`, rollup will automatically extract **named exports** from commonjs `exports`.